lib/backend/watchersyncer/watchersyncer_test.go

```diff
@@ -127,25 +172,36 @@ var _ = Describe("Test the backend datastore multi-watch syncer", func() {
 		// - watch succeeds ...
 		//
 		// The longest of these is resource 2 (since the watcher poll timer is longer).  We'll
-		// check that connection succeeds within +- 30% of the expected interval.
+		// check that connection succeeds within -30% and +50% of the expected interval.
 		By("Driving a bunch of List complete, Watch fail events for 2/3 resource types")
 		expectedDuration := watchersyncer.WatchPollInterval * 2
 		minDuration := 70 * expectedDuration / 100
-		maxDuration := 130 * expectedDuration / 100
+		maxDuration := 150 * expectedDuration / 100
+		before := time.Now()
 		rs.clientListResponse(r1, emptyList)
 		rs.ExpectStatusUpdate(api.ResyncInProgress)
 		rs.clientWatchResponse(r1, genError)
+		rs.ExpectStatusUnchanged()
```

Comment:
So I understand, were these test changes required? Or were they added to test the new functionality?

It seems like this might be covered in the new tests you added above, so wondering what this change is for.

Reply:
The `ExpectStatusUpdate` ones are required for any status change but you're right the `ExpectStatusUnchanged` calls aren't required and aren't needed here as that's tested above.